.github/workflows/lint.yml

```diff
@@ -10,7 +10,7 @@ on:
     paths:
       - '**.py'
       - 'pyproject.toml'
-      - 'requirements*.txt'
+      - 'requirements**.txt'
```

Comment:
Is this actually finding requirements.txt and requirements/*.txt?

Reply:
Yes. `**` glob includes `/` character.

Reply:
This also is currently working for the `e2e-nvidia-t4-x1.yml` job

Reply:
Thanks, glob doesn't always work that way:
```
glob.glob("requirements**.txt", recursive=True)
['requirements.txt', 'requirements-dev.txt', 'requirements-vllm-cuda.txt']
```
So wanted to double check.

Reply:
See https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

> 
`*`: Matches zero or more characters, but does not match the / character. For example, Octo* matches Octocat.
`**`: Matches zero or more of any character.